incubator/catalog-apicurio/src/main/java/io/aklivity/zilla/runtime/catalog/apicurio/internal/ApicurioCatalogHandler.java

```diff
@@ -69,56 +80,60 @@ public ApicurioCatalogHandler(
         this.client = HttpClient.newHttpClient();
         this.crc32c = new CRC32C();
         this.artifacts = new Int2ObjectCache<>(1, 1024, i -> {});
-        this.globalIds = new Int2ObjectCache<>(1, 1024, i -> {});
+        this.schemaIds = new Int2ObjectCache<>(1, 1024, i -> {});
         this.maxAgeMillis = config.maxAge.toMillis();
         this.groupId = config.groupId;
+        this.useId = config.useId;
+        this.useIdHeaderName = APICURIO_VALUE + useId;
+        this.idEncoding = config.idEncoding;
+        this.artifactPath = useId.equals(GLOBAL_ID) ? ARTIFACT_BY_GLOBAL_ID_PATH : ARTIFACT_BY_CONTENT_ID_PATH;
```

Comment:
Suggest you invert this logic, falling back to global id by default, since it is the actual default.